src/test/java/com/exasol/bucketfs/BucketTest.java

```diff
@@ -36,17 +36,39 @@ void testGetDefaultBucket() {
     // [itest->dsn~bucket-lists-its-contents~1]
     @Test
     void testListBucketContents() throws BucketAccessException, InterruptedException {
-        assertThat(container.getDefaultBucket().listContents("/"), hasItem(startsWith("EXAClusterOS")));
+        assertThat(container.getDefaultBucket().listContents(), hasItem(startsWith("EXAClusterOS")));
+    }
+
+    // [itest->dsn~bucket-lists-its-contents~1]
+    @Test
+    void testListBucketContentsWithRootPath() throws BucketAccessException, InterruptedException {
+        assertThat(container.getDefaultBucket().listContents(), hasItem(startsWith("EXAClusterOS")));
     }
```

Comment:
These are testing same functionality. Did you intended the first one include another path?

Reply:
Fixed.